docs/src/modules/utils/parseMarkdown.test.js

```diff
@@ -23,4 +23,43 @@ describe('parseMarkdown', () => {
       });
     });
   });
+
+  describe('prepareMarkdown', () => {
+    it('returns the table of contents with html and emojis stripped', () => {
+      const markdown = `
+# Support
+## Community help (free)
+### GitHub <img src="/static/images/logos/github.svg" width="24" height="24" alt="GitHub logo" loading="lazy" />
+### Unofficial 👍
+### Warning ⚠️
```

Comment:
Funnily enough the GitHub UI does not display this colored but only the text representation.

Reply:
Huh, now it works. Who knows..